### ISSUE TITLE
is it possible to add Flux w/loras and i2i?

### ISSUE THREAD

**Issue by KINGLIFER** [BetaDoggo/ComfyUI-Cloud-APIs#4]
xxxxxxxxxxx

**Comment by BetaDoggo** [comment]
Sure, here's an example workflow: 
![workflow](https://github.com/user-attachments/assets/60382672-f64b-424e-8d61-b50ba4e67943)
Make sure that you're using the newest version and that the dependencies are also all installed. The lora_url should be a direct download link (you can usually get this by right clicking the download button and selecting copy link). Also keep in mind that the required strength value varies a lot by the style of the image. Let me know if you have any issues.

**Comment by KINGLIFER** [comment]
Some reason when I use this the image comes out b/w. On the site it does not. 
Will do some more testing to see why that is.

**Comment by BetaDoggo** [comment]
@KINGLIFER I'm an idiot, I just realized that my image conversion code was also converting the image to greyscale. I copied it from a previous project but forgot to remove that part. This was also an issue in the vision model nodes. Thank you so much for helping me catch this. I've pushed the fix now so you'll have to update.

**Comment by KINGLIFER** [comment]
And it has been updated? If I turn down the denoise It still goes b/w on the site this is not the case.

**Comment by BetaDoggo** [comment]
Yes it was fixed in aadf07b9c37dc69c1905b62b9fc2b28f22fa8c19 
When I set the strength to 0.01 I get a nearly identical image (it's stretched because of the set generation resolution)
![image](https://github.com/user-attachments/assets/cb773ade-0643-48c9-a1a5-ee59a1b0c143)
What method are you using to update the extension?

**Comment by KINGLIFER** [comment]
Not sure what I am doing wrong because it is still B$W and I can not afford to do more so. Maybe I will try another time.
Thank you for the work  work.

**Comment by BetaDoggo** [comment]
Sorry I've been busy the last week. As far as a know the only cause of this would be that the version you have is out of date. You should try reinstalling the extension completely (deleting the folder) and trying this again. Otherwise I'm not sure if there's anything else I can do, I can't replicate this issue anymore on a clean install.

**Comment by KINGLIFER** [comment]
Totally understand the busy time and no need to apologize.  Curious did you try a low denoise like say .63? 

Thank you will reinstall.

**Comment by BetaDoggo** [comment]
In the example above I set my strength to 0.01 and it produced an almost identical image to the input (stretched to match the generation resolution). Here are two with the regular image2image node: 
![i2i](https://github.com/user-attachments/assets/d97a5783-195e-4615-9522-95fb198c6b24)
![dogi2i](https://github.com/user-attachments/assets/1aa7a09a-f446-4f54-b358-78620567b7b0)

**Comment by KINGLIFER** [comment]
Please add the option to have multiple images for Flux w/loras and i2i (fal). I do think you can go up to 4.

**Comment by BetaDoggo** [comment]
@KINGLIFER Sorry for the late response, as far as I can tell i2i only works with a single image, you're probably thinking of ipadapter, controlnet or one of the other model extensions that has gotten flux support in the last few months. I don't really use this project anymore so it's a pretty low priority, but I'll see if I can throw together a node based on the [flux-general](https://fal.ai/models/fal-ai/flux-general/playground) and [flux-general/image-to-image](https://fal.ai/models/fal-ai/flux-general/image-to-image/playground) endpoints this weekend.

**Comment by KINGLIFER** [comment]
No. I am talking about how many results you can get up to 4. Outputs not
inputs.

**Comment by KINGLIFER** [comment]
If you do general please add 4 outputs so I mean you can pick 1 to 4

**Comment by BetaDoggo** [comment]
Oh ok yeah that shouldn't be too hard, I should be able to add that to the existing nodes. Rather than adding 4 outputs to the node I'll probably make another node to split the output so that it's an optional feature.

**Comment by BetaDoggo** [comment]
I've just pushed support for multiple image outputs. It outputs as comfyui's normal multi-image format and can be split by a new node (SplitImages) into up to 4 individual images. I've added it to the falfluxAPI, falfluxi2iAPI, and falfluxloraAPI nodes. I might add it to all of the other image output nodes later but I don't feel like retesting every single one right now, let me know if there's a specific node you want it on.
![workflow with split](https://github.com/user-attachments/assets/f83377d1-feba-4690-a7d0-c08e54d95505)

I didn't find the time to make a flux/general node this weekend and I'm not sure if I'll do it just based on the pricing of the images (13/$1). I might still make nodes based on the redux endpoints since they're still relatively affordable, but that would have to wait until next weekend.

**Comment by KINGLIFER** [comment]
Boss do you have a donation link?
Thank you so much.

**Comment by KINGLIFER** [comment]
Hummm I wanted to update but I can not find ComfyUI-Cloud-APIs in my manager.
I assume the name has changed?

**Comment by BetaDoggo** [comment]
@KINGLIFER You might be running an outdated version of comfy manager. Comfy Manager switched to using the official ComfyUI Registry not too long ago. The list that the older versions are using is outdated. 

If you're running the newest version the search should look something like this:
![image](https://github.com/user-attachments/assets/6384407b-8ab7-48bf-84d9-11623183ccdd)

You also need to select the specific 1.0.5 version or it will complain about security settings:

![image](https://github.com/user-attachments/assets/45ecfaae-2243-4eb8-bd3a-c35f54e8c466)

**Comment by KINGLIFER** [comment]
Please add this node... fal-ai/veo2/image-to-video
Love it. I was hoping you could add this one as well. I know you put a lot of work into these things for free so I am saying thank you.

**Comment by KINGLIFER** [comment]
:(

**Comment by BetaDoggo** [comment]
Sorry I was busy last week and couldn't get to this until now. I don't think I'm going to add veo support due to the cost. It's $2.50 per 5 seconds so it would be very expensive to test properly. I think Google's pricing is absurd since I believe they also offer a certain number of free generations through their own AIstudio platform.

**Comment by KINGLIFER** [comment]
I have no problem testing for you in needed. Or is there another img to
video that's cheaper?

**Comment by KINGLIFER** [comment]
Ok it seems my offer is not something you would want so how about Kling-video? .09 seems better than .5 per second.

**Comment by BetaDoggo** [comment]
Sorry for the late response, I haven't been checking my email, I did end up giving a Veo2 node a shot after all, it doesn't have many options so I only needed to do a single test generation.

I'll be a bit more available the next few days, if there are any other specific video endpoints you want support for let me know. Most of the work is already done with the veo node, it just needs to modified a bit for other models with more settings.

![Image](https://github.com/user-attachments/assets/748fb0c0-8d1a-4756-9524-2457a66345f4)

The new version (1.1.0) should already be available on the comfy registry, you can update to the current version through comfy-manager.

![Image](https://github.com/user-attachments/assets/3eae5314-c15e-426e-8197-1a8413c75de4)
